R/tostring.R

```diff
@@ -105,9 +105,8 @@ setMethod("toString", "MatrixPrintForm", function(x,
                                                   col_gap = x$col_gap,
                                                   hsep = default_hsep()) {
   mat <- matrix_form(x, TRUE)
-  browser()
   inset <- table_inset(mat)
-
+  if(tf_wrap) browser()
```

Comment:
Need this removed before we can merge.

Reply:
I think I removed in commit [forgot browser](https://github.com/insightsengineering/formatters/pull/86/commits/a0e7bd8e4712caa59ea3fb6b8f968c005372d6b0)

but I do not understand the errors that are still blocking this PR

Reply:
I think the problem comes from the following... How did you solve the problem @gmbecker?

> So to conclude, this isn't me not liking that we're doing linting, or even the settings of the linter. This is the linter fundamentally not functioning properly within the GH action.


_Originally posted by @gmbecker in https://github.com/insightsengineering/formatters/issues/83#issuecomment-1379467728_